traitsui/examples/demo/Standard_Editors/tests/test_TextEditor_demo.py

```diff
@@ -0,0 +1,61 @@
+"""
+This example demonstrates how to test interacting with a textbox created
+using TextEditor.
+
+The GUI being tested is written in the demo under the same name (minus the
+preceding 'test') in the outer directory.
+"""
+
+import os
+import runpy
+import unittest
+
+# FIXME: Import from api instead
+from traitsui.testing.tester.command import KeyClick, KeySequence
+from traitsui.testing.tester.query import DisplayedText
+from traitsui.testing.tester.ui_tester import UITester
+
+#: Filename of the demo script
+FILENAME = "TextEditor_demo.py"
+
+#: Path of the demo script
+DEMO_PATH = os.path.join(os.path.dirname(__file__), "..", FILENAME)
+
+
+class TestTextEditorDemo(unittest.TestCase):
+
+    def test_run_demo(self):
+        demo = runpy.run_path(DEMO_PATH)["demo"]
```

Comment:
I only realized now that the `load_demo` in `test_all_examples` is uncalled for. It can be replaced by `runpy` from the standard library like this (or maybe it wasn't possible before for some other reasons, but it is now okay to use it.)